enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/MultiRealmAuthManager.java

```diff
@@ -187,6 +188,7 @@ public void shutdown() throws Throwable
                 ((RealmLifecycle) realm).shutdown();
             }
         }
+        securityLog.shutdown();
```

Comment:
Agreed, but as described in my other response below, I see the issue with the two SecurityLog constructors has orthogonal to this fix.